### DISCUSSION TITLE
How to disconnect client every two hours in pro version?

### DISCUSSION THREAD

**Discussion by akong77** [rustdesk/rustdesk-server#342] [General]
Hello,
How to disconnect client every two hours in pro version?Or have another solution?

**Comment by akong77** [comment]
Or if user idle then force disconnect?

**Comment by rustdesk** [comment]
In nightly build (it seems in 1.2.3 also, please double check), https://github.com/rustdesk/rustdesk/pull/5674

![image](https://github.com/rustdesk/rustdesk-server/assets/71636191/f6b77318-8747-4b9b-8b4b-d9a3be953ba5)

**Reply by akong77** [reply]
OK,Is it have set in pro version?
